proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -157,15 +158,43 @@ message MavenConfig {
   repeated RuntimeConfig additional_runtimes = 3;
 }
 
+message SwiftConfig {
+  // RuntimeLibrary describes the runtime library dependency of the generated code.
+  message RuntimeLibrary {
+    // The name of the runtime library package, github.com/apple/swift-protobuf.
+    string package = 1;
+    // The version of the runtime library, i.e. "1.21.0"
+    string version = 2;
+    // The products exported by the runtime library.
+    repeated string products = 3;
```

Comment:
I vaguely understand why we only need the names of the products here, but should we future-proof ourselves a bit and make this a `message Product` with a `string name = 1` and then make this `repeated Product products`? wdyt?

It would make me feel better if we needed to encode more into these in the future to have it be extensible - looking at something like https://github.com/apple/swift-protobuf/blob/main/Package.swift#L16-L33

Lastly, do we need this field to be set (should we validate it as non-empty)?

Reply:
That's a great idea 🙏🏼 

This field doesn't need to be set. If the plugin source packages don't specify any platform version information, it can be omitted.